rules-tests/Php81/Rector/FuncCall/NullToStrictStringFuncCallArgRector/Fixture/skip_error_type.php.inc

```diff
@@ -1,27 +0,0 @@
-<?php
-
-namespace Rector\Tests\Php80\Rector\FuncCall\NullToStrictStringFuncCallArgRector\Fixture;
-
-use Nette\Utils\Strings;
-
-final class SkipErrorType
```

Comment:
This fixture removal need to be reverted, per my comment before, ArrayDimFetch without context got ErrorType, it will make the rule unusable as it change the valid type string already, which mostly correct on regex check usage